tests/test_utils/get_all_tests.py

```diff
@@ -56,7 +56,7 @@ def print_all_cases(xunit_test_file_path):
     modules = set()
 
     for test_case in test_cases:
-        the_module = ".".join(test_case.get("classname").split(".")[:-1])
+        the_module = test_case.get("classname").rpartition(".")[0]
```

Comment:
Hmm this should be

```suggestion
        the_module = test_case["classname"].rpartition(".")[0]
```

But it’s a problem inherited from old code.